### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by JoesKeyboard**
What do you think peter

### CODE REVIEW

**Comment by JoesKeyboard** [comment]
Well done buddy

File: README.md [hunk 1]
```diff
@@ -1,5 +1,3 @@
 "# hello-world" 
-#Peter Davis Sucks
-#Peter Davis Sucks More
-#Peter Davis Sucks Even More
+#Welcome to the Party
 #Word to your mother.
```
Comment (davispetere) [inline README.md:3]: this is what we do

**Comment by JoesKeyboard** [comment]
Ya it is

File: README.md (hunk 1 above)
Comment (JoesKeyboard) [reply README.md:3]: ok

**Pull request merged**